tests/Test.py

```diff
@@ -17,3 +17,16 @@ class TestAccountCreation:
     assert createPet.tags[0]['id'] == 1
     assert createPet.tags[0]['name'] == "FirstName"
     assert createPet.status == "available"
+
+    pet_id=createPet.id
+    verification_name=createPet.name
+    print(pet_id)
+
+    retrieved_pet=PetAPIService().get_pet_by_id(pet_id).convert_response_to_pet_model()
```

Comment:
move convert_response_to_pet_model() to PetAPIService.py level for not writing it each time with get_pet_by_id